packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -256,7 +256,14 @@ class AnalyzeCommand extends FlutterCommand {
       mainFile.path
     ];
 
-    printStatus(cmd.join(' '));
+    if (argResults['congratulate']) {
```

Comment:
Makes sense. I'd like to try and limit the total number of config flags in general however. What about emitting the prefix and suffix (`Analyzing foo...` and `No analyzer warnings`) by default, and having a flag like `--quiet` to disable them? It's nice to have the feedback for users - probably the more common case - and build systems or whatnot that just want the messages can use the flag.